stdlib/public/runtime/Errors.cpp

```diff
@@ -18,18 +18,24 @@
 #include <stdlib.h>
 #include <string.h>
 #include <stdint.h>
+#if defined(_MSC_VER)
+#include <io.h>
+#else
 #include <unistd.h>
+#endif
+#if !defined(_MSC_VER)
 #include <pthread.h>
```

Comment:
I don't think we are using pthread for anything in this file, could you try removing the include?